### ISSUE TITLE
OpenStreetMaps API vs GoogleMaps API

### ISSUE THREAD

**Issue by KarolDawidziuk** [pawelstepien/pathfinder-server#4]
We should choose one of API or check if there are some alternatives. 

Pros of GoogleMaps API:
- good documentation,
- easy to use,
- a lot of features out of the box,
- 200$ to use
Cons:
- credit card added required.

OpenStreetMaps API:
Pros:
- there is no requirement for credit card
Cons:
- request limits ( I'm not sure how much ) per minute/hour,
- documentation is pretty poor

**Comment by danieldegroot2** [comment]
Hey @KarolDawidziuk,

You can find more information here:
- [_Switch2OSM_](https://switch2osm.org/)
- The [Policies](https://operations.osmfoundation.org/policies/) on the Operations Working Group (OWG) website. Their contact details are on the home page.

tldr;
- You should preferably use a (commercial) tile provider (Leaflet providers are partially free).
- Otherwise, the application will simply be blocked if it is deemed to use too many resources. ('heavy use'); refer to the above.
- (Bulk downloading; there is an [open issue](https://github.com/openstreetmap/owg-website/issues/66) to clarify the bulk downloading limit specified in the Tiles Usage Policy.)
